python/torch_mlir/dialects/torch/importer/jit_ir/CMakeLists.txt

```diff
@@ -4,11 +4,12 @@
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 include(TorchMLIRPyTorch)
-TorchMLIRProbeForPyTorchInstall()
-find_package(Torch 1.8 REQUIRED)
+set(Torch_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../libtorch/share/cmake/Torch")
```

Comment:
So we will always require manually building libtorch by the side? I.e. we won't support working from installed packages anymore?

Reply:
By default we will download the latest libtorch and unpack it in the root dir. If a user wants to use any other variant of a released or older libtorch they can unzip it in the root dir and we will respect that and avoid downloading the latest.

Reply:
And can you make there be a good diagnostic if the user doesn't have that directory populated? Something that gives specific instructions for how to rebuild/download it.